templates/install_rvm.sh.erb

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+USER=<%= @user %>
+
+SERVERS=(
+    hkp://keys.gnupg.net
+    hkp://pgp.mit.edu
+)
+
+for server in "${SERVERS[@]}"
+do
+    su -c "gpg --keyserver ${server} --recv-keys D39DC0E3" - $USER
```

Comment:
You could also just include the GPG key itself if there's some kind of network problem that's preventing the downloading.  Up to you.  

(RVM's gpg signing is still of dubious value anyway if the script we're getting from `https://get.rvm.io` isn't signed itself. Nothing's stopping the @TheNSABackdoorCommittee from doing a MITM attack there.)